pkger/README.md

```diff
@@ -0,0 +1,639 @@
+# Pkger: the What and How
+
+Responsibilities
+
+* Translating a declarative [package](#anatomy_of_a_package) file (either JSON | Yaml | Jsonnet) into resources in the platform
+* Exporting existing resources in the form of a pkg (either JSON | Yaml)
+* Managing the state of a pkg's side effects via a stack
+
+## Anatomy of a package
+
+A package is a collection of resource configurations. 
+These resource configurations can be seen in full in the [pkger/testdata](https://github.com/influxdata/influxdb/tree/master/pkger/testdata) directory.
+The package itself does have any state. 
+Packages may contain resources that are not uniquely identifiable within the platform and some that are not.
```

Comment:
Packages may contain resources that are ~not~ uniquely identifiable within the platform and some that are not.